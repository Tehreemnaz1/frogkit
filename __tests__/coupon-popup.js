import { mount, shallow } from 'vue-test-utils'
import Vue from 'vue'
import Dialogue from '@/components/dialogue'
import CouponPopup from '@/components/coupon-popup'

const propsData = {
  visible: true,
  dark: true,
  requiredAction: false,
  lang: 'de',
  title: 'Hiergeblieben!',
  text: 'Smartfrog schenkt dir den ersten Monat!',
  codeText: 'Gutscheincode:',
  code: 'FREE1',
  btnText: 'Jetzt einlÃ¶sen',
  noText: 'Jetzt nicht'
}

describe('Popup component', () => {
  it('should render component and match snapshot', () => {
    const wrapper = mount(CouponPopup, { propsData })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
