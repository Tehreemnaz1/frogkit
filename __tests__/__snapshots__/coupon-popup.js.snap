// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup component should render component and match snapshot 1`] = `<dialogue visible="visible" dark="true"></dialogue>`;
